Vector.sc

```diff
@@ -213,16 +250,20 @@ RealVector3D[slot] : RealVector {
 
 	phi { ^atan2(this[2], (this[0].squared + this[1].squared).sqrt) }
 
+	azi {
```

Comment:
Shouldn't azimuth be arctan( y / x ) ?

See [wikipedia](https://en.wikipedia.org/wiki/Spherical_coordinate_system#Coordinate_system_conversions)

Reply:
Yes, I can do it that way to keep things consistent. I just through of turning the entire 3D space on its side and doing the same operation about the x-axis as opposed to the z-axis.